File: 프로그래머스/unrated/181927. 마지막 두 원소/마지막 두 원소.cs
using System;

public class Solution {
    public int[] solution(int[] num_list) 
    {
        int[] answer = new int[num_list.Length + 1];
        Array.Copy(num_list, answer, num_list.Length);
        int a = num_list[num_list.Length - 2];
        int b = num_list[num_list.Length - 1];
        answer[answer.Length - 1] = b > a ? b - a : b * 2;
        return answer;
    }
}

File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> num_list) {
    vector<int> answer = vector<int>(num_list);
    
    int last = num_list[num_list.size() - 1];
    int preLast = num_list[num_list.size() - 2];
    
    int newItem = last > preLast ? last - preLast : last * 2;
    answer.push_back(newItem);
        
    return answer;
}